locales/en/public.json

```diff
@@ -79,7 +79,7 @@
     "MAXIMUM_SIZE": "Maximum size ({{unit}})",
     "SAVE_CHANGES": "Save changes",
     "TEMPLATE_HELPER_TEXT": "The Event Template to be applied to Automated Analysis recordings.",
-    "TEMPLATE_INVALID_WARNING": "WARNING: Setting a Target Template as a default template type configuration may not apply to all Target JVMs if the JVMs do not support them."
+    "TEMPLATE_INVALID_WARNING": "WARNING: Setting the Target Template as a default template type configuration may not apply to all Target JVMs if the JVMs do not support them."
```

Comment:
I'm not sure about this change. It's describing event templates of the kind "Target", as opposed to "Custom":

![image](https://github.com/cryostatio/cryostat-web/assets/3787464/e011a769-f016-4485-bd76-59da3452dd72)

There are usually at least two such Target Templates for any given target application.

This warning is to say that if you pick a Target Template, then this only makes sense if every other Target has that same kind of Template (matching by name).

To give a concrete example of such an invalid configuration:

![image](https://github.com/cryostatio/cryostat-web/assets/3787464/936582f8-d64e-485a-a58f-0c7f4722820e)

I could select the Cryostat event template here, but any configuration using this template (Automated Analysis card, Automated Rule) would not be able to apply this event template to another target which doesn't have that same template:

![image](https://github.com/cryostatio/cryostat-web/assets/3787464/80f77086-a8fd-4ecb-a610-411389c5c67b)

Reply:
Ok I'll keep it how it is :+1: